src/transform-applier.js

```diff
@@ -505,14 +505,42 @@ const _parseUrl = (value, windowRef) => {
  */
 const transformStrokeWidths = function (svgTag, windowRef, bboxForTesting) {
     const inherited = Matrix.identity();
-    const applyTransforms = (element, matrix, strokeWidth, fill) => {
+
+    const makeTransformedGradient = (gradientId, element, matrix) => {
+        const doc = windowRef.document;
+        // Need path bounds to transform gradient
+        const svgSpot = doc.createElement('span');
+        let bbox;
+        if (bboxForTesting) {
+            bbox = bboxForTesting;
+        } else {
+            try {
+                doc.body.appendChild(svgSpot);
+                const svg = SvgElement.set(windowRef.document.createElementNS(SvgElement.svg, 'svg'));
+                const path = SvgElement.set(windowRef.document.createElementNS(SvgElement.svg, 'path'));
```

Comment:
```suggestion
                const svg = SvgElement.set(doc.createElementNS(SvgElement.svg, 'svg'));
                const path = SvgElement.set(doc.createElementNS(SvgElement.svg, 'path'));
```
Should I include this code style enhancement as part of this PR?

Reply:
Sure